### PR TITLE
provide a UNIX export suitable show

### PR DESCRIPTION
**Pull request by esc**
This allows doing things like:

  $ eval $(afp --show account)

to setup AWS credentials in your environment.

### CODE REVIEW

**Comment by mriehl** [comment]
Semantically I'd expect `show` to just output the credentials and nothing more. Consider s/show/show-export/g?

**Comment by esc** [comment]
Well, on Windows, `show` does a `set BLA='FOO'` so this behaviour would be consistent.

I'd propose to make `--show` just show and `--export` to use `set` and `export` on windows and unix respectively.

**Comment by mriehl** [comment]
:+1: for `show` and `export`

**Comment by esc** [comment]
FWIW: I plan to tackle this tomorrow.

**Comment by esc** [comment]
This is ready for review and merge.

File: README.rst [hunk 1]
```diff
@@ -112,3 +112,52 @@ Specify the URL of the AFP server, overriding any config file:
...
+   AWS_VALID_SECONDS='600'
+   AWS_SESSION_TOKEN='XXX'
+   AWS_SECURITY_TOKEN='XXX'
+   AWS_SECRET_ACCESS_KEY='XXX'
+   AWS_EXPIRATION_DATE='1970-01-01T01:00:00Z'
+   AWS_ACCESS_KEY_ID='XXX'
+
+.. code-block:: console
+
+   $ afp --export myrole
```
Comment (tvollmer-at-is24) [inline README.rst]: Missing account name in command line

File: README.rst [hunk 2]
```diff
@@ -112,3 +112,52 @@ Specify the URL of the AFP server, overriding any config file:
...
+bash, you can use ``--show`` or ``--export`` to display the credentials. You
+can use the usual UNIX tools to add/remove them from your environment.
+``--show`` will just show them and ``--export`` will show them in format
+suitable for an export into your environment, i.e. prefixed with ``export`` for
+UNIX and ``set`` for Windows.
+
+
+.. code-block:: console
+
+   $ afp --show myrole
```
Comment (tvollmer-at-is24) [inline README.rst]: Missing account name in command line

File: README.rst [hunk 3]
```diff
@@ -112,3 +112,52 @@ Specify the URL of the AFP server, overriding any config file:
...
+
+
+The following examples work in zsh, to add and remove them from your
+environment:
+
+Adding credentials:
+
+.. code-block:: console
+
+   $ eval $(afp --export <rolename>)
```
Comment (tvollmer-at-is24) [inline README.rst]: Missing account name in command line

**Comment by esc** [comment]
Alright thanks for review @tvollmer-at-is24 -- letting the checks run through and then merging.

**Comment by esc** [comment]
here goes

**Comment by esc** [comment]
@tvollmer-at-is24 think we should communicate this internally?

**Comment by esc** [comment]
I bumped the version on PyPi too!

**Pull request merged**